front/src/components/App.tsx

```diff
@@ -12,6 +12,8 @@ import SearchPage from 'pages/SearchPage';
 import CalendarPage from 'pages/CalendarPage';
 import MyShowsPage from 'pages/MyShowsPage';
 import { gAnalyticsID } from 'utils/constants';
+import ProtectedRoute from './ProtectedRoute';
```

Comment:
Could we move ProtectedRoute to `components/common/ProtectedRoute`? (will have to create a common folder)

Reply:
hmn.... will do that